root/usr/share/gpoint/lib/nmea.lua

```diff
@@ -225,7 +226,8 @@ local function getGSA(GnssData, resp)
 		"id12",       -- ID of 12th satellite used for fix
 		"pdop",       -- PDOP
 		"hdop",       -- HDOP
-		"vdop"        -- VDOP
+		"vdop",       -- VDOP
+		"sc",         -- checksum
```

Comment:
Why are the control amounts needed in this particular data? The comma after the checksum is not needed

Reply:
Actally , the checksum is use to avoid null array,  the data in RM500Q is

Reply:
and comma , more like a ctrl-c and ctrl-v problem :-p , i should fix it , sorry

Reply:
The same data is sent by ep06 and other modems that I had for testing, the fact is that the checksum in the function is checked by the code:

-- message parsing, checksum checking
local function getCropData(data, msg)

And in the future it was not added to the data because the UI does not work with it))) I would like to clarify if you had any errors if you did not use a checksum in the data? Thanks!

Reply:
The function "doTable" make key and value match to their field, And if it have value and the key is NULL , lua will report 'nil from table' error. So I add this key to make it no happend.

Reply:
Everything is clear, thank you again for your contribution to the project!